README.md

```diff
@@ -53,6 +53,10 @@ selecting the `Use DiCy` option in the settings page. [DiCy][] will be installed
 automatically and so no further action is required for either TeX Live or
 MiKTeX.
 
+### Code Highlighting
+In order to properly work in Atom with this package, you'll need a code
+highlighting extension, namely [`language-latex`][language-latex-pkg].
+
```

Comment:
I would definitely add the `language-r` mention. I don't think it we have to explain that `language-knitr` imports `language-r` (and `language-latex`) since knitr users will understand that R is required.

```md
Additionally, you might want to consider installing
[`language-knitr`][language-knitr] and [`language-r`][language-r]
if you want to take advantage of the provided knitr support.
```